node/src/actor_system_factory.rs

```diff
@@ -1787,8 +1775,8 @@ mod tests {
         };
     }
 
-    fn check_pool_bind_message(recording: &Arc<Mutex<Recording>>) {
-        let _pool_bind_message = Recording::get::<PoolBindMessage>(recording, 1);
+    fn check_pool_bind_message(recording: &Arc<Mutex<Recording>>, idx: usize) {
```

Comment:
When I saw this the first time from outside the function it was confusing because I falsely thought the number meant a number of messages expected. I know, it's stupid, if I think of it a bit deeper but yet the immediate impression makes me say that the numbers might be confusing in some way.

I don't think you need to necessarily find a better way, and so this can be accepted, because I would feel myself uneasy about such a request if you were me and I received it. I just let my thoughts be here.   
(making these assert helpers clearer could be done by well named enums as the argument(s) but risking being cumbersome after all again)

Reply:
I renamed some functions. Thanks for the heads-up: I think they're much clearer now.